src/popup/components/BrowserPopup.tsx

```diff
@@ -152,43 +147,6 @@ export default class BrowserPopup extends React.Component<Partial<IBrowserPopupP
    * This method of communication with content scripts in fact proves cumbersome;
    * we should move more and more to the global redux store
    */
-  handleAnnotationModeClick = (e) => {
```

Comment:
I haven't thought about removing those parts of application, but we should indeed and that's kind of nice bonus ;)